doc/src/manual/faq.md

```diff
@@ -106,40 +106,21 @@ file.jl` to execute a script while being able to catch
 
 ### How do I pass options to `julia` using `#!/usr/bin/env`?
 
-Passing options to `julia` in so-called shebang by, e.g.,
-`#!/usr/bin/env julia --startup-file=no` may not work in some
-platforms such as Linux.  This is because argument parsing in shebang
-is platform-dependent and not well-specified.  In a Unix-like
-environment, a reliable way to pass options to `julia` in an
-executable script would be to start the script as a `bash` script and
-use `exec` to replace the process to `julia`:
+Passing options to `julia` in a so-called shebang line, as in
+`#!/usr/bin/env julia --startup-file=no`, will not work on many
+platforms (BSD, macOS, Linux) where the kernel, unlike the shell, does
+not split arguments at space characters. The option `env -S`, which
+splits a single argument string into multiple arguments at spaces,
+similar to a shell, offers a simple workaround:
 
 ```julia
-#!/bin/bash
-#=
-exec julia --color=yes --startup-file=no "${BASH_SOURCE[0]}" "$@"
-=#
-
+#!/usr/bin/env -S julia --color=yes --startup-file=no
 @show ARGS  # put any Julia code here
 ```
 
-In the example above, the code between `#=` and `=#` is run as a `bash`
-script.  Julia ignores this part since it is a multi-line comment for
-Julia.  The Julia code after `=#` is ignored by `bash` since it stops
-parsing the file once it reaches to the `exec` statement.
-
 !!! note
-    In order to [catch CTRL-C](@ref catch-ctrl-c) in the script you can use
```

Comment:
This doesn't apply any more?

Reply:
It still does, however after switching to `env -S` [this note](https://docs.julialang.org/en/v1/manual/faq/#How-do-I-pass-options-to-julia-using-#!/usr/bin/env?) would just say

>In order to [catch CTRL-C](https://docs.julialang.org/en/v1/manual/faq/#catch-ctrl-c) in the script you can use
>```julia
>#!/usr/bin/env -S julia -e 'include(popfirst!(ARGS))'
>@show ARGS  # put any Julia code here
>```
>instead. Note that with this strategy [PROGRAM_FILE](https://docs.julialang.org/en/v1/base/constants/#Base.PROGRAM_FILE) will not be set.

Apart from the final sentence about `PROGRAM_FILE` not being set, it seems redundant, as it just applies exactly what [the previous question](https://docs.julialang.org/en/v1/manual/faq/#catch-ctrl-c) said. I'll move that to the previous question, which uses the same `-e 'include(popfirst!(ARGS))'` trick that causes `PROGRAM_FILE` not to be set.

Reply:
Final sentence of deleted redundant note now moved to previous question.